File: src/model/Manager.ts
export class Manager {
    public id: string;
    public name: string;
    public role: string;

    constructor() {
        this.id = "";
        this.name = "";
        this.role = "";
    }

    toJSON() {
        return { "id" : this.id != undefined ? this.id : "", "name" : this.name != undefined ? this.name : "", "role" : this.role != undefined ? this.role : "" };
    }

    copy(manager: Manager) {
        this.id = manager.id;
        this.name = manager.name;
        this.role = manager.role;
    }
}

File: Admin/src/main/java/com/safetyBread/admin/model/Manager.java
package com.safetyBread.admin.model;

import lombok.Getter;
import lombok.Setter;

@Getter @Setter
public class Manager {
	private String id;
	private String name;
	private String role;
	
	@Override
	public String toString() {
		return "{\"id\": \"" + id + "\", \"name\" : \"" + name + "\", \"role\": \"" + role + "\"}";
	}
}
